### PR TITLE
Feature/tri 296 deployment

### PR DESCRIPTION
**Pull request by ds-jkreutzfeld**

### CODE REVIEW

File: irs-api/src/main/java/net/catenax/irs/controllers/IrsController.java [hunk 1]
```diff
@@ -114,10 +115,10 @@ public JobHandle getBomLifecycleByGlobalAssetId(final @Valid @ParameterObject Ir
                                     }),
     })
     @GetMapping("/jobs/{jobId}")
-    public ResponseEntity<Jobs> getBOMForJobId(
+    public Jobs getBOMForJobId(
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/net/catenax/irs/controllers/IrsController.java:118]: Name is getBOMForJobId but returns Jobs. Will this be refactored in [TRI-365](https://jira.catena-x.net/browse/TRI-365)?

File: irs-api/src/main/java/net/catenax/irs/services/IrsPartsTreeQueryService.java [hunk 2]
```diff
@@ -29,12 +55,28 @@
...
-        //return null;
+        final String uuid = request.getGlobalAssetId().substring(IrsApiConstants.URN_PREFIX_SIZE);
+        final var params = Map.of(AASRecursiveJobHandler.ROOT_ITEM_ID_KEY, uuid);
+        final JobInitiateResponse jobInitiateResponse = orchestrator.startJob(params);
+
+        if (jobInitiateResponse.getStatus().equals(ResponseStatus.OK)) {
+            final String jobId = jobInitiateResponse.getJobId();
+            return JobHandle.builder().jobId(UUID.fromString(jobId)).build();
+        } else {
+            // TODO (jkreutzfeld) Improve with better response (proper exception for error responses?)
```
Comment (ds-jhartmann) [inline irs-api/src/main/java/net/catenax/irs/services/IrsPartsTreeQueryService.java:77]: Pending TODO

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]
1 open TODO, 1 question. The rest looks good to me.

File: irs-api/src/main/java/net/catenax/irs/controllers/IrsController.java (hunk 1 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/net/catenax/irs/controllers/IrsController.java:118]: Yes :)

File: irs-api/src/main/java/net/catenax/irs/services/IrsPartsTreeQueryService.java (hunk 2 above)
Comment (ds-jkreutzfeld) [reply irs-api/src/main/java/net/catenax/irs/services/IrsPartsTreeQueryService.java:79]: Yeah, this was out of scope of the story (just deployment). I added some basic implementation here to actually get a request working.

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**